packages/rich-text/src/test/index.native.js

```diff
@@ -264,9 +262,7 @@ describe( '<RichText/>', () => {
 			// Assert.
 			expect( screen.toJSON() ).toMatchSnapshot();
 		} );
-	} );
 
-	describe( 'Line height', () => {
```

Comment:
Should we remove test cases for the line height? I know we won't parse it from a CSS value but we're still parsing the value and [enforcing a minimum value](https://github.com/WordPress/gutenberg/pull/47284/files#diff-7af18df9c3e07e67518894550a2e1d9d627feb879a2003a2e231ef927d4d06f8R1011), WDYT?